packages/apps/patterns/react-ui/src/panels/JoinPanel/view-states/InvitationAuthenticator.tsx

```diff
@@ -15,7 +15,7 @@ import { ViewState, ViewStateHeading, ViewStateProps } from './ViewState';
 const pinLength = 6;
 
 export interface InvitationAuthenticatorProps extends ViewStateProps {
-  Domain: 'Space' | 'Halo';
+  Kind: 'Space' | 'Halo';
```

Comment:
maybe factor out KIND as a type? since used in multiple places
it doesn't save code, but might help connect the two halves with common defs file that might be a seed?
free to ignore.